### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by tracychms**
# PR into Azure/azureml-previews

## Checklist

I have:

- [ ] read and followed the contributing guidelines

## Changes

-

Fixes #

### CODE REVIEW

File: previews/online-endpoints-model-profiler/README.md [hunk 1]
```diff
@@ -30,54 +52,217 @@ Follow the example in this [tutorial](how-to-deploy-managed-online-endpoints.md)
...
-| `environment_variables.TARGET_RPS` | string | Target reuest per second for the benchmarking tool. Supported by `wrk2` and `labench`. |  | `50` |
-| `environment_variables.CLIENTS` | string | Number of clients for the benchmarking tool. Default value is set to be the number of `request_settings.max_concurrent_requests_per_instance`, or `WORKER_COUNT` in environment variables in online deployment, or 1 if both two values are not set or Null. Supported by `labench`. |  |  |
-| `environment_variables.TIMEOUT` | string | Timeout in seconds for each request. Supported by `labench`. |  | `10s` |
-| `inputs.payload` | string | The file of sampled payload containing a list of JSON objects that is stored in an AML registered datastore. |  |  |
-
-#### Create a profiling job
+| --- | ----- | ----------- | -------------- | ------------- |
+| `command` | string | The command for running the profiling job. | `python -m online_endpoints_model_profiler ${{inputs.payload}}` | - |
+| `experiment_name` | string | The experiment name of the profiling job. An experiment is a group of jobs. | - | `Downloads` |
```
Comment (Zhong-J) [inline previews/online-endpoints-model-profiler/README.md:135]: default value is "downloads"?

File: previews/online-endpoints-model-profiler/README.md [hunk 2]
```diff
@@ -30,54 +52,217 @@ Follow the example in this [tutorial](how-to-deploy-managed-online-endpoints.md)
...
-* Create proper role assignment for accessing online endpoint resources.
+  ```bash
+  az ml compute create --name $PROFILER_COMPUTE_NAME --size $PROFILER_COMPUTE_SIZE --identity-type SystemAssigned --type amlcompute
+  ```
 
-:::code language="azurecli" source="https://github.com/tracychms/profiling/blob/main/code/profiling/create-profiling-compute.sh" ID="create_compute_cluster_for_hosting_the_profiler" :::
+* Create proper role assignment for accessing online endpoint resources. The compute needs to have contributor role to the machine learning workspace.
 
-:::code language="azurecli" source="https://github.com/tracychms/profiling/blob/main/code/profiling/create-profiling-compute.sh" ID="create role assignment for acessing workspace resources" :::
+  ```bash
```
Comment (Zhong-J) [inline previews/online-endpoints-model-profiler/README.md:86]: The role assignment is so complicated from my perspective. Can we give a more straightforward way, and also put a role assignment official document here for reference?

File: previews/online-endpoints-model-profiler/README.md [hunk 3]
```diff
@@ -1,26 +1,48 @@
 # Online Endpoints Model Profiler
```
Comment (Zhong-J) [inline previews/online-endpoints-model-profiler/README.md:1]: I woud advice to have a real example put in "example" folder, covering online endpoint create, model deployment, compute creation to endpointpoint profiling, which is easier for user to try from scratch. And in the readme, we can go through how to use the feature from e2e.

File: previews/online-endpoints-model-profiler/README.md (hunk 1 above)
Comment (xy223) [reply previews/online-endpoints-model-profiler/README.md:135]: This is the default value set by aml jobs, which i would find confusing as well. but I did an experiment, If I don't set experiment name, the experiment name would be Downloads

File: previews/online-endpoints-model-profiler/README.md (hunk 2 above)
Comment (xy223) [reply previews/online-endpoints-model-profiler/README.md:86]: official doc attached.

File: previews/online-endpoints-model-profiler/README.md (hunk 3 above)
Comment (xy223) [reply previews/online-endpoints-model-profiler/README.md:1]: Added a link to the example profiling script.

**Review by Zhong-J — APPROVED** [review]

**Pull request merged**